### PR TITLE
:sparkles: polishing report responsitivity

### PR DESCRIPTION
**Pull request by MatissJanis**
Related https://github.com/actualbudget/actual/issues/3282

### CODE REVIEW

File: packages/desktop-client/src/components/common/Tooltip.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export const Tooltip = ({
 
   return (
     <View
-      style={{ minHeight: 'auto' }}
+      style={{ minHeight: 'auto', flexShrink: 0 }}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/common/Tooltip.tsx:55]: Screenshot 2024-10-11 at 19 47 09

fix for the tiny icons

File: packages/desktop-client/src/components/filters/FilterExpression.tsx [hunk 2]
```diff
@@ -62,8 +62,21 @@ export function FilterExpression<T extends RuleConditionEntity>({
         variant="bare"
         isDisabled={customName != null}
         onPress={() => setEditing(true)}
+        style={{
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/filters/FilterExpression.tsx:65]: Screenshot 2024-10-11 at 19 49 20

fix for filters expanding out-of-bounds

File: packages/desktop-client/src/components/filters/FiltersButton.tsx [hunk 3]
```diff
@@ -6,7 +6,10 @@ import { Button } from '../common/Button2';
 export function FiltersButton({ onPress }: { onPress: () => void }) {
   return (
     <Button variant="bare" onPress={onPress}>
-      <SvgFilter style={{ width: 12, height: 12, marginRight: 5 }} /> Filter
+      <SvgFilter
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/filters/FiltersButton.tsx:9]: Fix the the icon in "Filters" button becoming small when resizing browser

File: packages/desktop-client/src/components/reports/Header.tsx [hunk 4]
```diff
@@ -66,82 +67,68 @@ export function Header({
     <View
       style={{
         padding: 20,
-        paddingTop: 0,
         flexShrink: 0,
       }}
     >
-      <View
+      <SpaceBetween
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/reports/Header.tsx:73]: Various small re-alignments in cashflow & networth pages on small devices (i.e. mobile). To make the filters look slightly better on small resolutions.

File: packages/desktop-client/src/components/reports/ReportSidebar.tsx [hunk 5]
```diff
@@ -177,16 +178,13 @@ export function ReportSidebar({
             <strong>Display</strong>
           </Text>
         </View>
-        <View
+        <SpaceBetween
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/reports/ReportSidebar.tsx:181]: No user-facing update here. Just a small code improvement.

File: packages/desktop-client/src/components/reports/reports/Spending.tsx [hunk 6]
```diff
@@ -23,6 +23,7 @@ import { Block } from '../../common/Block';
 import { Button } from '../../common/Button2';
 import { Paragraph } from '../../common/Paragraph';
 import { Select } from '../../common/Select';
+import { SpaceBetween } from '../../common/SpaceBetween';
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/reports/reports/Spending.tsx:26]: Improving how the filters look in mobile & smaller resolutions.

**Comment by MikesGlitch** [comment]
Found one small thing but it's unrelated to your PR. If you're interested: 
test

**Review by MikesGlitch — APPROVED** [review]
Looks great.

**Comment by MatissJanis** [comment]
Good idea @MikesGlitch . Updated the report topbar to scale better. It's not _ideal_.. but better than before.

**Review by MikesGlitch — APPROVED** [review]

**Pull request merged**